### PR TITLE
IA-2609: update link to user guide

### PR DESCRIPTION
**Pull request by quang-le**
Now we have a website , we should link to it

Related JIRA tickets : IA-2609

## Self proofreading checklist

- [ ] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes
- Replace link to google doc with link to (web) user guide on readthedocs

## How to test

Explain how to test your PR.
If a specific config is required explain it here (dataset, account, profile, ...)

## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/88ff290c-9ce5-48a1-8948-b7871bee4b00

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/app/components/SidebarMenuComponent.js [hunk 1]
```diff
@@ -87,14 +87,10 @@ const styles = theme => ({
     },
 });
 
-const localizedManualUrl = (locale, account) => {
-    if (locale === 'fr' && account.name === 'RDC') {
-        return 'https://docs.google.com/document/d/1lKyhbKDLZpHtAsf3K6pRs0_EAXWdSDsL76Ohv0cyZQc/edit';
-    }
-
-    return account.user_manual_path
```
Comment (hakifran) [inline hat/assets/js/apps/Iaso/domains/app/components/SidebarMenuComponent.js:95]: @quang-le, the user_manual_path will not be used anymore? if yes why don't we just remove it from the account?

Comment (hakifran) [reply hat/assets/js/apps/Iaso/domains/app/components/SidebarMenuComponent.js:95]: And on the line 200, the current.account parameter should be removed from the call of **localizedManualUrl**. 
Screenshot from 2024-01-12 09-43-13

Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/app/components/SidebarMenuComponent.js:95]: Mmmh, good comment. I guess, We should keep the thing configurable indeed.

Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/app/components/SidebarMenuComponent.js:95]: I've restored it and asked for an updated link to the rdc doc

**Review by hakifran — APPROVED** [review]
Good for me

**Pull request merged**